.github/actions/install-dependencies/action.yml

```diff
@@ -1,82 +1,56 @@
-name: install-dependencies
+name: Install Dependencies
 
 inputs:
-  dependency-identifiers:
-    type: string
+  msbuild-required-tools-target:
+    description: msbuild target to call to get a list of required external tools
+    default: GetRequiredExternalTools
+  required-tools:
+    description: external tools to install, in addition to the ones returned by `msbuild-required-tools-target`
   msbuild-dependency-target:
-    type: string
+    description: msbuild target to install dependencies
   solution-file-path:
-    type: string
+    description: path to the .sln file
   ckan-compatible-versions:
-    type: string
+    description: KSP versions for CKAN to be compatible with
     default: |
       1.12
       1.11
       1.10
       1.9
       1.8
-  ckan-filters:
```

Comment:
this variable does not appear to be used so I killed it

Reply:
It was used at some point….